### PR TITLE
One typo in markdown and one bug in solver

### PR DESCRIPTION
**Pull request by msb002**
I noticed a couple of bugs in this while running it on my own. Im guessing you intend to use these for future classes so I thought it was worth passing this along.

### CODE REVIEW

**Comment by kyleniemeyer** [comment]
@msb002 Thanks! can you summarize the changes? unfortunately git doesn't make changes to Jupyter notebooks very easy to read

**Comment by msb002** [comment]
Sure no problem!

There was a missing minus sign on the last equation (cell 5), and the function being solved is defined as "equilibrium" but the code had "get_equilibrium" in the solver (cell 7).

**Comment by amit1rrr** [comment]
> unfortunately git doesn't make changes to Jupyter notebooks very easy to read

Absolutely. You might want to checkout [ReviewNB](https://www.reviewnb.com/), code review tool for notebooks. Provides visual diffs & commenting for Jupyter notebooks on GitHub. Completely free for open source repositories.

Disclaimer: I built ReviewNB

diff_and_commenting

**Comment by kyleniemeyer** [comment]
Thanks for catching these @msb002, I made these changes directly
